docs/en/operations/system-tables/query_log.md

```diff
@@ -151,6 +152,7 @@ query_kind:
 databases:                             []
 tables:                                []
 columns:                               []
+columns_in_where:                      ['type']
```

Comment:
Seems weird that `type` appears in field `columns_in_where` but not in `columns`.